storage/src/test/java/org/apache/kafka/server/log/remote/metadata/storage/TopicBasedRemoteLogMetadataManagerTest.java

```diff
@@ -171,111 +146,121 @@ private void waitUntilConsumerCatchesUp(TopicIdPartition newLeaderTopicIdPartiti
         long time = System.currentTimeMillis();
 
         while (true) {
-            if (System.currentTimeMillis() - time > timeoutMs) {
-                throw new TimeoutException("Timed out after " + timeoutMs + "ms ");
+            if (System.currentTimeMillis() - time > 30000L) {
+                throw new TimeoutException("Timed out after " + 30000L + "ms ");
```

Comment:
nit: Let's make it as a variable, so that if we want to change the timeout, we can change one place only.